site/tests/migrator/transform/test_record_transform.py

```diff
@@ -136,6 +136,28 @@ def zenodo_record_data():
                     }
                 ],
             },
+            "custom": {
+                # dwc
+                "dwc:collectionCode": ["MUSM", "ZMPC"],
```

Comment:
Note: I have added tests for the most common fields that I found among several taxonomic treatments in Zenodo. Is it worth adding for every single field since the logic is the same?

Reply:
if the logic is the same, probably not. However if you want the migration to be airtight, I would add it for the peace of mind :)